packages/jest-circus/src/legacy_code_todo_rewrite/jest_adapter.js

```diff
@@ -77,6 +77,12 @@ const jestAdapter = async (
     runtime.requireModule(config.setupTestFrameworkScriptFile);
   }
 
+  if (config.setupTestsAfterJest.length) {
```

Comment:
Hmm.. you bring up a good point! I lean more towards option 2 since both serve the same purpose. I can't think of a case where I would have both set in my configuration.

With that, another idea I'd like to suggest is that if both are set, we could log a warning that `setupTestsAfterJest` is taking precedence, and that `setupTestFrameworkScriptFile` is being ignored.

Reply:
Just curious, how's the behaviour like if you pass in a wrong path to a file? Is that error handled or is that handled by the runtime. It would be good to gracefully handle that scenario

Reply:
What about changing line 77 to:

```js
config.setupTestsAfterJest.push(config.setupTestFrameworkScriptFile);
```

Reply:
@rickhanlonii We would need to include info in the docs that mentions the module set using `setupTestFrameworkScriptFile` will run after the code set using the new configuration. If I do that, should we remove the original implementation of `setupTestFrameworkScriptFile`? This would handle all use cases.

Reply:
Yeah agreed, can probably add it to the doocs here https://jestjs.io/docs/en/configuration#setuptestframeworkscriptfile-string

What's the original implementation? Do you mean just line 77?